### DISCUSSION TITLE
Is it possible to run DBusWorld alongside truckersmp-cli?

### DISCUSSION THREAD

**Discussion by BeeBee1234567891** [truckersmp-cli/truckersmp-cli#313] [Q&A] [answered]
Is it possible to run [DBus](https://dbusworld.com/) inside of a proton or wine instance when running truckersmp-cli?

**Comment by Lucki** [comment, marked as answer]
You can specify programs to start alongside TruckersMP in the [configuration file](https://github.com/truckersmp-cli/truckersmp-cli#configuration-file).

Don't forget to have a look at the fully commented example configuration file: https://github.com/truckersmp-cli/truckersmp-cli#example-configuration-file

If dbusworld runs inside wine then it should be possible.

**Comment by BeeBee1234567891** [comment]
Thanks for the reply. I've managed to get the app launching, but am stuck in the setup process within the app. It asks you to find and select the game folder, but when navigating through explorer, there is no sign of the game files and all steamapp folders are empty.
I'm guessing this is security-related, but is there any known way to bypass this to get the game files showing for this app?

![image](https://user-images.githubusercontent.com/33805937/186271224-a95a303f-263a-4413-99b7-2f73f1648cf6.png)
![image](https://user-images.githubusercontent.com/33805937/186271285-a3dd2807-9cf9-456f-adee-a3ef5cd53d5c.png)

**Reply by Lucki** [reply]
Your files are in `Z:\`. Press the `+`.
